### ISSUE TITLE
Toggle additional transformations not working

### ISSUE THREAD

**Issue by rufhausen** [dericofilho/sublime-phpfmt#136]
I noticed today that phpfmt was no longer working, and in troubleshooting noticed that when I attempt to access "toggle additional transformations" from the command palette, I see the contents of the attached screenshot instead of the transformations, so I'm wondering if that's the problem. I've tried removing and re-installing the plugin both manually and thru package control to no avail.
![screen_shot_2015-06-10_at_11_17_01_am](https://cloud.githubusercontent.com/assets/43862/8089053/b6fdac0a-0f62-11e5-9598-be5875bf4644.png)

For reference, this is my current config, though it's not working either when I start over with an empty config:
{
	"autocomplete": false,
	"autoimport": false,
	"enable_auto_align": true,
	"excludes":
	[
		"WordWrap",
		"LeftWordWrap"
	],
	"format_on_save": true,
	"ignore_list":
	[
		".blade.php"
	],
	"indent_with_space": 4,
	"laravel_style": false,
	"passes":
	[
		"MergeNamespaceWithOpenTag",
		"RemoveUseLeadingSlash",
		"SpaceBetweenMethods",
		"PrettyPrintDocBlocks",
		"GeneratePHPDoc",
		"AlignDoubleArrow",
		"AlignEquals"
	],
	"php55warning": true,
	"php_bin": "/Applications/MAMP/bin/php/php5.6.7/bin/php",
	"psr1": false,
	"psr2": true,
	"version": 3
}

Thanks.

**Comment by dericofilho** [comment]
@rufhausen I am really sorry for this bug. I am going to fix it ASAP.

**Comment by dericofilho** [comment]
@rufhausen please, manually upgrade to https://github.com/dericofilho/sublime-phpfmt/tree/3.24.0

**Comment by rufhausen** [comment]
I did the upgrade and that does fix the problem with additional transformations. However, I've traced the issue of the plugin back to the ignore_list entry in the config(you had suggested this in response to my comment in issue #131. Saving a file with "ignore_list" returns the following in the console:
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "/Users/gtaylor/Library/Application Support/Sublime Text 3/Packages/phpfmt/phpfmt.py", line 1350, in run
    src = dofmt(self, self.view, None, src)
  File "/Users/gtaylor/Library/Application Support/Sublime Text 3/Packages/phpfmt/phpfmt.py", line 162, in dofmt
    ignore_list = ignore_list.split(" ")
AttributeError: 'list' object has no attribute 'split'

Thanks!
